### ISSUE TITLE
Inconsistencies in Deprecate

### ISSUE THREAD

**Issue by zimeon** [LD4/entity_metadata_management#53]
In my attempt to address #40 with #52, I think I see inconsistency with how Deprecate is used.

In https://docs.google.com/document/d/1eiFANJvR6cYE3Tx3cTsLhDO_BxZFr7NKPrQnBPh48rk/edit# there are two scenarios:

Scenario 1 - just update entity (replacement already exists)
```
{ 
  "type": "Deprecate",
  "orderedItems": [ {
        "type": "Update",
        "object": { "type": "Term", "id": "dep#1" }
  } ]
}
```

Scenario 2 - Need to create replacement as well as update deprecated
```
{ 
  "type": "Deprecate",
  "orderedItems": [ {
        "type": "Create",
        "object": { "type": "Term", "id": "new#2" }
  }, {
        "type": "Update",
        "object": { "type": "Term", "id": "dep#1" }
  } ]
}
```
and these two are consistent with each other and with other parts of https://docs.google.com/document/d/1eiFANJvR6cYE3Tx3cTsLhDO_BxZFr7NKPrQnBPh48rk/edit#

However, in the current examples for [Split](https://ld4.github.io/entity_metadata_management/api/0.1/recommendations.html#split-entity) and [Merge](https://ld4.github.io/entity_metadata_management/api/0.1/recommendations.html#merge-entity), we have `Deprecate` at an atomic level. E.g. in Split:
```
{ 
  "type": "Split",
  "orderedItems": [ {
        "type": "Create",
        "object": { "type": "Term", "id": "new#2" }
  }, {
        "type": "Create",
        "object": { "type": "Term", "id": "new#3" }
  }, {
        "type": "Deprecate",    <----- THIS IS NOT CONSISTENT WITH SCENARIO 1 ABOVE
        "object": { "type": "Term", "id": "dep#1" }
  } ]
}
```

I do like the atomic form of `Deprecate` because it provides additional information over `Update` but with no more complexity.

**Comment by kefo** [comment]
I would be down with `Deprecate` being usable at the atomic level because it would address my issues with `Delete`.  See:  https://github.com/LD4/entity_metadata_management/issues/32#issuecomment-1243888530

**Comment by zimeon** [comment]
This has been addressed by discussion in #32, and changes to clarify `Deprecate`/`Delete` in https://github.com/LD4/entity_metadata_management/pull/78
